### PR TITLE
Added step for adding  Operator attributes for filtering

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-1041

Preview build: https://console-filters--ocpdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Received a :+1: from Rob in chat

@yapei @jianzhangbjz Can you please provide QE review?

**Comment by ahardin-rh** [comment]
@adellape Just an FYI on this update

**Review by sheriff-rh — APPROVED** [review]
LGTM!

File: modules/olm-enabling-operator-restricted-network.adoc [hunk 1]
```diff
@@ -88,3 +88,15 @@ spec:
 <1> Inject the images referenced by the Operator via environment variables.
 <2> Specify each image by a digest (SHA), not by an image tag.
 <3> Also reference the Operator container image by a digest (SHA), not by an image tag.
+
+* Look for the `Disconnected` annotation, which indicates that the Operator works
```
Comment (yapei) [inline modules/olm-enabling-operator-restricted-network.adoc:92]:
```
Look for the `Disconnected` annotation
```

Am I misunderstanding? I think it should be `Look for the `operators.openshift.io/infrastructure-features` annotation`

Comment (robszumski) [reply modules/olm-enabling-operator-restricted-network.adoc:92]: That annotation will be used for a number of different features. This doc is about disconnected, so the only relevant value is `disconnected`.

Comment (yapei) [reply modules/olm-enabling-operator-restricted-network.adoc:92]: Thanks for the explanation, sounds reasonable

**Pull request merged**